controllers/storagecluster/job_templates.go

```diff
@@ -57,6 +58,20 @@ func (obj *ocsJobTemplates) ensureCreated(r *StorageClusterReconciler, sc *ocsv1
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return reconcile.Result{}, fmt.Errorf("failed to create Template : %v", err.Error())
 		}
+
+		// Delete a template job after the job gets completed
```

Comment:
the order of sequence should be like below right?

```
job:=Get()
if err!=nil && !NotPresent{
return nil,err
}
if error==nil && job.Status.Succeeded >= 1{
// delete it and return
} 
err=CreateOrUpdate()
return err
```

Reply:
Hi Madhu, actually no. While creating we create the openshift templatest only. When a user wants they create the job by using the oc process command. 

```
oc process -n openshift-storage ocs-osd-removal -p FAILED_OSD_IDS=0 FORCE_OSD_REMOVAL=true |oc create -n openshift-storage -f -
```

Reply:
Got it, Thank you